code/espurna/broker.h

```diff
@@ -49,7 +50,10 @@ TBrokerCallbacks<TArgs...> TBroker<type, TArgs...>::callbacks;
 // --- Some known types. Bind them here to avoid .ino screwing with order ---
 
 using StatusBroker = TBroker<TBrokerType::STATUS, const String&, unsigned char, unsigned int>;
-using SensorBroker = TBroker<TBrokerType::SENSOR, const String&, unsigned char, double, const char*>;
+
+using SensorReadBroker = TBroker<TBrokerType::SENSOR_READ, const String&, unsigned char, double, const char*>;
+using SensorReportBroker = TBroker<TBrokerType::SENSOR_REPORT, const String&, unsigned char, double, const char*>;
```

Comment:
A note to the reader:
The very first arg is needed to distinguish between two otherwise identical templates, without it both ReadBroker and ReportBroker would be the exact same class.